AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Storage/StorageEngine.swift

```diff
@@ -11,14 +11,47 @@ import Foundation
 import AWSPluginsCore
 
 final class StorageEngine: StorageEngineBehavior {
-
     // TODO: Make this private once we get a mutation flow that passes the type of mutation as needed
     let storageAdapter: StorageEngineAdapter
 
     private var syncEngine: RemoteSyncEngineBehavior?
 
     private weak var api: APICategoryGraphQLBehavior?
 
+    var iStorageEnginePublisher: Any?
+
+    var iCancellable: Any?
+    @available(iOS 13.0, *)
+    var cancellable: AnyCancellable? {
+        get {
+            if let iCancellable = iCancellable {
+                return (iCancellable as! AnyCancellable) //swiftlint:disable:this force_cast
```

Comment:
Same here? Or make iCancellable to `AnyCancellable`?

Reply:
Updated, thanks!